apps/api-extractor/src/aedoc/test/Token.test.ts

```diff
@@ -42,14 +38,14 @@ describe('Token tests', function (): void {
       } catch (error) {
         errorThrown = true;
       }
-      assert.equal(errorThrown, false);
```

Comment:
You can use toThrow to test error thrown.

Reply:
Good point

---
In reply to: [152471857](https://github.com/Microsoft/web-build-tools/pull/450#discussion_r152471857) [](ancestors = 152471857)